### PR TITLE
docs: DD-523 Wallets > Wallet Workflows (Overview)

### PR DESCRIPTION
**Pull request by jennifertrin**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [X] Follows the [developer docs style guide](style-guide.md).
- [X] Follows the [best practices and guidelines](README.md#best-practices).
- [X] New documentation pages include [document tags](README.md#document-tags).
- [X] New documentation pages include [SEO keywords](README#seo-keywords).
- [X] New documents are in the `.mdx` file format to support the previous two components. 
- [X] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+keywords: [beginner, concept, wallet, wallets, authentication, assets]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Wallet Overview
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:7]:
```suggestion
# Wallet overview
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+This document provides an overview of the purpose of different types of wallets on the Internet Computer and how to interact with them. 
+
+Users of the Internet Computer use wallets for the following activities:
+
+- Hold, transfer, and receive assets such as fungible including ICP and other ICRC1/ICRC2 tokens (ex. ckBTC, ckETH) and non-fungible tokens.
+- Deploy, upgrade, and perform operations on canisters.
+
+Decentralized applications developed on ICP can authenticate users using wallets or [Internet Identity](/docs/current/developer-docs/identity/internet-identity/overview).
+
+Though not directly considered a wallet, Internet Identity behind the scenes generates a principal. This principal not only identifies you to various user-facing dapps but is also an identifier to send and receive assets on ICP.
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:20]:
```suggestion
Users and developers on ICP use wallets for the following activities:

- Holding, transferring, and receiving assets such as fungible including ICP and other ICRC1/ICRC2 tokens (ex. ckBTC, ckETH) and non-fungible tokens.
- Deploying, upgrading, and performing operations on canisters.
- Authenticating with decentralized applications.

On other networks, these activities may all use the same wallet account and workflow. On ICP, each of these activities will likely use a separate wallet address or service. This document provides an overview of each different workflow, the type of account associated with it, and how to interact with it.
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Types of Wallets
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:22]:
```suggestion
## Types of wallets 
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- Hold, transfer, and receive assets such as fungible including ICP and other ICRC1/ICRC2 tokens (ex. ckBTC, ckETH) and non-fungible tokens.
+- Deploy, upgrade, and perform operations on canisters.
+
+Decentralized applications developed on ICP can authenticate users using wallets or [Internet Identity](/docs/current/developer-docs/identity/internet-identity/overview).
+
+Though not directly considered a wallet, Internet Identity behind the scenes generates a principal. This principal not only identifies you to various user-facing dapps but is also an identifier to send and receive assets on ICP.
+
+## Types of Wallets 
+
+On ICP, users may use different types of wallets for different types of interactions. Here is a table that describes the types of wallets based on interaction:
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:24]:
```suggestion
On ICP, both developers and end users will use different types of wallets for different interactions. Here is a table that describes each type of wallet and the activities associated with them: 
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+ <table>
+    <tr>
+        <th>Type of Wallet</th>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:28]:
```suggestion
        <th>Type of wallet</th>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        <th>Activities</th>
+        <th>Examples</th>
+    </tr>
+    <tr>
+        <td>[Asset Wallet](/overview)</td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:33]:
```suggestion
        <td>[Asset wallet](/overview.mdx)</td>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        <td> - Hold, transfer, and receive assets such as fungible tokens including ICP and other ICRC1/ICRC2 tokens (ex. ckBTC, ckETH) and non-fungible tokens 
+        <br>
+        - View token holdings
+        <br>
+        - Stake ICP and create a neuron to participate in the NNS
+        <br>
+        - Can also be used for user based authentication
+        </td>
+        <td> - Plug, Stoic, Oisy. Please refer to the [Asset Wallets docs for the full list](/docs/current/developer-docs/defi/wallets/overview#third-party-custody) </td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:42]:
```suggestion
        <td> - Hold, transfer, and receive assets such as fungible tokens including ICP and other ICRC1/ICRC2 tokens (ex. ckBTC, ckETH) and non-fungible tokens.
        <br>
        - View token holdings.
        <br>
        - Stake ICP and create a neuron to participate in the NNS.
        <br>
        - User authentication.
        </td>
        <td> - Plug, Stoic, Oisy. Please refer to the [asset wallets docs for the full list](/docs/current/developer-docs/defi/wallets/overview#third-party-custody) </td>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    </tr>
+    <tr>
+        <td>[Canisters](/docs/current/developer-docs/smart-contracts/overview/introduction)</td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:45]:
```suggestion
        <td>[Canister-controlled wallet](/docs/current/developer-docs/smart-contracts/overview/introduction)</td>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        <td>Using [Chain Fusion](/docs/current/developer-docs/multi-chain/overview), canisters on ICP can hold, sign, and transact with assets on various chains including ICP, Bitcoin, Ethereum, EVMs, and more.
+        </td>
+        <td>Any canister</td>
+    </tr>
+    <tr>
+        <td>[Cycles](//docs/current/developer-docs/defi/cycles/cycles-wallet) (will be depreciated from dfx) </td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:51]:
```suggestion
        <td>[Cycles wallet](//docs/current/developer-docs/defi/cycles/cycles-wallet) (will be depreciated from dfx) </td>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+        <td> - Holds cycles needed to deploy, upgrade, and perform other operations on canisters
+        <br>
+        - The Cycles Wallet will be replaced by the Cycles ledger. By using the Cycles ledger, cycles will compile to common ICRC1/ICRC2/ICRC3 standards.
+        </td>
+        <td>The Cycles Wallet will be replaced by the [Cycles ledger](docs/current/developer-docs/defi/cycles/cycles-ledger). By using the Cycles ledger, cycles will compile to common ICRC1/ICRC2/ICRC3 standards.</td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:56]:
```suggestion
        <td> - Holds cycles needed to deploy, upgrade, and perform other operations on canisters.
        <br>
        - The cycles wallet is still supported, but has been replaced by the [cycles ledger](docs/current/developer-docs/defi/cycles/cycles-ledger). It is recommended to use the cycles ledger instead.
        </td>
```

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+    </tr>
+</table>
+
+### Other authentication methods
+
+As mentioned, Internet Identity is not directly considered a wallet but more of an authentication method. However, an Internet Identity is tied to a unique principal for each unique domain that can be used to hold and receive assets.
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:62]:
```suggestion
Internet Identity is not directly considered a wallet but more of an authentication method. However, an Internet Identity is tied to a unique principal for each unique domain that can be used to hold and receive assets. 
```

File: docs/developer-docs/defi/wallets/workflow.mdx (hunk 2 above)
Comment (jennifertrin) [reply docs/developer-docs/defi/wallets/workflow.mdx:20]: Thank you for this wording. This definitely makes it more clear to developers and users from other ecosystems.

File: docs/developer-docs/defi/wallets/workflow.mdx (hunk 7 above)
Comment (jennifertrin) [reply docs/developer-docs/defi/wallets/workflow.mdx]: Commit 2969d025d

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/defi/wallets/workflow.mdx [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+On ICP, both developers and end users will use different types of wallets for different interactions. Here is a table that describes each type of wallet and the activities associated with them: 
+
+ <table>
+    <tr>
+        <th>Type of wallet</th>
+        <th>Activities</th>
+        <th>Examples</th>
+    </tr>
+    <tr>
+        <td>[Asset wallet](/overview.mdx)</td>
```
Comment (jessiemongeon1) [inline docs/developer-docs/defi/wallets/workflow.mdx:30]:
```suggestion
        <td>[Asset wallet](overview.mdx)</td>
```

**Pull request merged**